src/main/resources/application.properties

```diff
@@ -15,6 +15,5 @@ auth.token.public.key=${PUBLIC_KEY}
 server.port=8082
```

Comment:
Don't hardcode a port in the properties file. You can override this in the Docker while running the application.